### PR TITLE
aws, aws-quicksight: add French translation

### PR DESCRIPTION
**Pull request by Nico385412**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

## Worries about this translation

I had some troubles to find a good translation, and i would be glad if another french can review my PR

The sentence is `Use auto prompt to help`. `prompt` is really hard to translate in french because there is no real direct translation, most of the time we must use a word which match the context.

I had a look at the AWS documentation and I notice that this flag just add auto-completion so, i translated `Utilise l'aide automatique au remplissage pour une commande` which means `Use the autocompletion help for this command`

I would like your point of view, is it ok ? (for native english speaker if it's ok to focus on the auto-completion)

### CODE REVIEW

**Comment by kbdharun** [comment]
@mcombeau can you review this PR and assist @Nico385412? 
Thanks in advance.

**Review by mcombeau — CHANGES_REQUESTED** [review]
It would be my pleasure, @kbdharun!

"Prompt" is a difficult word to translate in French and it seems our usual go-to term for it ("invite des commandes") would not make sense in this case since it strongly implies inviting the user to actually type a command. I have no hands-on experience with aws. That said, the [documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-parameters-prompting.html) about the CLI auto-prompt feature does describe it as an autocompletion tool rather than a prompt awaiting user input.

I think @Nico385412's compromise, focusing on the autocompletion aspect, is the smart choice here. Great work!

**Comment by kbdharun** [comment]
> It would be my pleasure, @kbdharun!
> 
> "Prompt" is a difficult word to translate in French and it seems our usual go-to term for it ("invite des commandes") would not make sense in this case since it strongly implies inviting the user to actually type a command. I have no hands-on experience with aws. That said, the [documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-parameters-prompting.html) about the CLI auto-prompt feature does describe it as an autocompletion tool rather than a prompt awaiting user input.
> 
> I think @Nico385412's compromise, focusing on the autocompletion aspect, is the smart choice here. Great work!

Thanks for the review 😄 .

File: pages.fr/common/aws-quicksight.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- Liste les tableaux de bords :
+
+`aws quicksight list-dashboards --aws-account-id {{id_compte_aws}}`
+
+- Affiche les informations détaillées sur un dataset :
+
+`aws quicksight describe-data-set --aws-account-id {{id_compte_aws}} --data-set-id {{id_data_set}}`
+
+- Affiche les personnes qui peuvent accéder au dataset et quels actions ils peuvent effectuer sur ce dernier :
```
[the remark this answers is not in the document]
Comment (Nico385412) [reply pages.fr/common/aws-quicksight.md:27]: damn thank you !

File: pages.fr/common/aws.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Récupère l'identité de l'appelant (utilisé pour débogguer les permissions) :
+
+`aws sts get-caller-identity`
+
+- Liste les ressources AWS d'une région et affiche le résultat en YAML :
+
+`aws dynamodb list-tables --region {{us-east-1}} --output yaml`
+
+- Utilise l'aide automatique au remplissage pour une commande :
```
[the remark this answers is not in the document]
Comment (Nico385412) [reply pages.fr/common/aws.md:27]: sounds better thanks !

**Comment by Nico385412** [comment]
> It would be my pleasure, @kbdharun!
> 
> "Prompt" is a difficult word to translate in French and it seems our usual go-to term for it ("invite des commandes") would not make sense in this case since it strongly implies inviting the user to actually type a command. I have no hands-on experience with aws. That said, the [documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-parameters-prompting.html) about the CLI auto-prompt feature does describe it as an autocompletion tool rather than a prompt awaiting user input.
> 
> I think @Nico385412's compromise, focusing on the autocompletion aspect, is the smart choice here. Great work!

Thanks ! i just added little changes as you asked. it should be good for now on

**Review by mcombeau — APPROVED** [review]
LGTM, thank you!

**Review by kbdharun — APPROVED** [review]
LGTM

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**